beacon-chain/verification/blob.go

```diff
@@ -44,12 +44,31 @@ var GossipSidecarRequirements = []Requirement{
 	RequireSidecarProposerExpected,
 }
 
+// InitsyncSidecarRequirements is the list of verification.Requirements to be used by the init-sync service
+// for batch-mode syncing. Because we only perform batch verification as part of the IsDataAvailable method
+// for blobs after the block has been verified, and the blobs to be verified are keyed in the cache by the
+// block root, it is safe to skip the following verifications.
+// RequireValidProposerSignature
```

Comment:
It should be removed from this list - I mistakenly added it to the list of skipped checks because I briefly forgot that the block root doesn't integrate the signature, so the signature still needs to be explicitly checked.